Rainfall-prediction/datacleaning.py

```diff
@@ -0,0 +1,23 @@
+# importing libraries
+import pandas as pd
+import numpy as np
```

Comment:
You can remove this line as it's not used anywhere.

Reply:
i will do that

Reply:
@Ashlin21dev only `import numpy as np`